app/controllers/csv_preview_redirect_controller.rb

```diff
@@ -0,0 +1,23 @@
+class CsvPreviewRedirectController < ApplicationController
+  before_action { expires_in(1.day, public: true) }
+
+  def redirect
+    host = if served_from_draft_asset_host?
+             # PLEK_HOSTNAME_PREFIX is set on draft environments,
+             # which means a 'draft-' prefix will be added to this host
+             Plek.find("origin", external: true)
+           else
+             Plek.find("www", external: true)
+           end
```

Comment:
In the live stack:
- Integration environment `Plek.find("www", external: true)` returns: https://www.integration.publishing.service.gov.uk 👍 
- Staging environment: `Plek.find("www", external: true)` returns: https://www.staging.publishing.service.gov.uk 👍
- But in production it returns https://www.publishing.service.gov.uk/ and I think we'd want it to return https://www.gov.uk

I think we might want to just fetch the [website_root](https://github.com/alphagov/plek/blob/3933b6a4d999984b58d2bd4b971e03b33e6588f0/lib/plek.rb#L115) instead: `Plek.website_root`

Reply:
Ohh I totally missed this one. Thank you 🙌